tools/topology/topology2/production/tplg-targets-ace2.cmake

```diff
@@ -17,6 +17,10 @@ SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,SDW_DMIC_STREAM=Capture-SmartMic,\
 SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 
+"cavs-sdw\;sof-lnl-cs42l43-l0\;PLATFORM=lnl,SDW_DMIC=1,NUM_HDMIS=3,NUM_SDW_AMP_LINKS=1,\
```

Comment:
Do we need to set NUM_HDMIS=3?

I don't know why it's needed, isn't there a default already?
Also we need to clean-up the ones with NUM_HDMIS=0.

Reply:
> Do we need to set NUM_HDMIS=3?
> 
> I don't know why it's needed, isn't there a default already? Also we need to clean-up the ones with NUM_HDMIS=0.

Yes, 3 is the default value of NUM_HDMIS, removed. Do you mean remove all NUM_HDMIS=0 for LNL like we did for MTL https://github.com/thesofproject/sof/pull/8789?

Reply:
yes, the NUM_HDMIS was a work-around when HDMI was missing upstream patches, but I don't think it's still the case on LNL, is it?
@kv2019i I've obviously lost the plot here.